editor/plugins/texture_3d_editor_plugin.cpp

```diff
@@ -160,14 +187,21 @@ Texture3DEditor::Texture3DEditor() {
 	layer->connect("value_changed", callable_mp(this, &Texture3DEditor::_layer_changed));
 
 	info = memnew(Label);
+
+	info->add_theme_color_override("font_color", Color::named("white"));
+	info->add_theme_color_override("font_shadow_color", Color::named("black"));
+
+	info->add_theme_font_size_override("font_size", 14 * EDSCALE);
+	info->add_theme_color_override("font_outline_color", Color::named("black"));
```

Comment:
It's preferable to use `Color(1, 1, 1)` and `Color(0, 0, 0)` as it leads to slightly smaller binary sizes from my testing.